demisto_sdk/commands/common/hook_validations/content_entity_with_test_playbooks_validator.py

```diff
@@ -0,0 +1,42 @@
+from abc import ABC
+
+from demisto_sdk.commands.common.hook_validations.content_entity_validator import \
+    ContentEntityValidator
+from demisto_sdk.commands.common.hook_validations.structure import \
+    StructureValidator
+from demisto_sdk.commands.common.hook_validations.test_playbook import \
+    TestPlaybookValidator
+
+
+class ContentEntityWithTestPlaybooksValidator(ContentEntityValidator, ABC):
+    def has_unskipped_test_playbook(self, id_set_file: dict, test_playbook_ids: list = []) -> bool:
+        """Check if the content entity has at least one unskipped test playbook."""
+        test_playbooks_unskip_status = {}
+
+        if type(self.current_file.get('tests')) is list:
+            test_playbook_ids.extend(self.current_file.get('tests', []))
```

Comment:
From what I gather this takes the `tests` straight from the entity - unfortunately, we stopped updating that as it is auto-updated in the build process - the correct way to check this is through the `conf.json` file in content repo.